spec/mailers/previews/candidate_mailer_preview.rb

```diff
@@ -889,4 +909,45 @@ def reasons_for_rejection_with_qualifications
       qualifications_which_qualifications: %w[no_english_gcse other],
     }
   end
+
+  def rejection_reasons
```

Comment:
could we add previews for RBD feedback received?

Reply:
Have amended the mailer preview methods commit https://github.com/DFE-Digital/apply-for-teacher-training/pull/6746/files#diff-89b5483a28529b4b09048423167740ede78ebf50538e5efa6299c00efd7d5e4fR551-R571